src/services/consent/utils/consent.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+	defineConsentStatus,
```

Comment:
Codacy found an issue: ['defineConsentStatus' is not defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)